src/App.tsx

```diff
@@ -37,12 +37,27 @@ const App = () => {
 
 	const backgroundColor = urlParams.get('bg') || process.env.DESCOPE_BG_COLOR;
 
+	const isWideContainer =
+		urlParams.get('wide') === 'true' || flowId === 'saml-config';
+
+	const containerClassName = isWideContainer ? 'descope-wide-container' : '';
+
+	const flowProps = {
+		flowId,
+		debug,
+		tenant: tenantId,
+		...(isWideContainer && { autoFocus: false })
+	};
+
 	return (
 		<AuthProvider projectId={projectId} baseUrl={baseUrl}>
 			<div className="app" style={{ backgroundColor }}>
 				{projectId && flowId ? (
-					<div className="descope-container" data-testid="descope-component">
-						<Descope flowId={flowId} debug={debug} tenant={tenantId} />
+					<div
+						className={`descope-base-container ${containerClassName}`}
```

Comment:
better to use `classnames` or `clsx` for this purpose, the extra `` if `containerClassName` is empty is something we should not have